From 1d697bcf2916a0c12517ce8e8b2aea4482b96a52 Mon Sep 17 00:00:00 2001
Message-ID: <1d697bcf2916a0c12517ce8e8b2aea4482b96a52.1697678664.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1697678664.git.sharlatanus@gmail.com>
References: <cover.1697678664.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Thu, 19 Oct 2023 00:31:49 +0100
Subject: [PATCH 09/19] gnu: go-github-com-tdewolff-test: Move to (gnu packages
 golang-check).

* gnu/packages/golang-check.scm (go-github-com-tdewolff-test):
Move from here...
* gnu/packages/golang.scm: ... to here.
---
 gnu/packages/golang-check.scm | 25 +++++++++++++++++++++++++
 gnu/packages/golang.scm       | 23 -----------------------
 2 files changed, 25 insertions(+), 23 deletions(-)

diff --git a/gnu/packages/golang-check.scm b/gnu/packages/golang-check.scm
index 4b7ca74cc2..fe442bdb8b 100644
--- a/gnu/packages/golang-check.scm
+++ b/gnu/packages/golang-check.scm
@@ -7,6 +7,7 @@
 ;;; Copyright © 2021 Tobias Geerinckx-Rice <me@tobias.gr>
 ;;; Copyright © 2022 Efraim Flashner <efraim@flashner.co.il>
 ;;; Copyright © 2023 Felix Lechner <felix.lechner@lease-up.com>
+;;; Copyright © 2023 Hilton Chain <hako@ultrarare.space>
 ;;;
 ;;; This file is part of GNU Guix.
 ;;;
@@ -157,6 +158,30 @@ (define-public go-github-com-stretchr-testify
 @end itemize")
     (license license:expat)))
 
+(define-public go-github-com-tdewolff-test
+  (package
+    (name "go-github-com-tdewolff-test")
+    (version "1.0.9")
+    (source
+     (origin
+       (method git-fetch)
+       (uri (git-reference
+             (url "https://github.com/tdewolff/test")
+             (commit (string-append "v" version))))
+       (file-name (git-file-name name version))
+       (sha256
+        (base32
+         "10myz3zdkqmx37cvj507h7l2ncb0rq9shqvz9ggq1swijbsvazff"))))
+    (build-system go-build-system)
+    (arguments
+     (list #:import-path "github.com/tdewolff/test"))
+    (home-page "https://github.com/tdewolff/test")
+    (synopsis "Go test helper functions")
+    (description
+     "This package implements a few functions that are useful for io testing,
+such as readers and writers that fail after N consecutive reads/writes.")
+    (license license:expat)))
+
 (define-public go-gopkg-in-check-v1
   (package
     (name "go-gopkg-in-check-v1")
diff --git a/gnu/packages/golang.scm b/gnu/packages/golang.scm
index c3cb30c344..effe3e7854 100644
--- a/gnu/packages/golang.scm
+++ b/gnu/packages/golang.scm
@@ -3646,29 +3646,6 @@ (define-public go-github-com-tdewolff-parse-v2
      "This package contains several lexers and parsers written in Go.")
     (license license:expat)))
 
-(define-public go-github-com-tdewolff-test
-  (package
-    (name "go-github-com-tdewolff-test")
-    (version "1.0.9")
-    (source (origin
-              (method git-fetch)
-              (uri (git-reference
-                    (url "https://github.com/tdewolff/test")
-                    (commit (string-append "v" version))))
-              (file-name (git-file-name name version))
-              (sha256
-               (base32
-                "10myz3zdkqmx37cvj507h7l2ncb0rq9shqvz9ggq1swijbsvazff"))))
-    (build-system go-build-system)
-    (arguments
-     (list #:import-path "github.com/tdewolff/test"))
-    (home-page "https://github.com/tdewolff/test")
-    (synopsis "Go test helper functions")
-    (description
-     "This package implements a few functions that are useful for io testing,
-such as readers and writers that fail after N consecutive reads/writes.")
-    (license license:expat)))
-
 (define-public go-github-com-tj-docopt
   (package
     (name "go-github-com-tj-docopt")
-- 
2.41.0

